### PR TITLE
match child module datums to parent module

### PR DESCRIPTION
**Pull request by snopoke**
Slightly hacky workaround to making sure entry session datum ids are consistent between parent and child modules.

The crux of the feature is here: https://github.com/dimagi/commcare-hq/compare/child-module-suite?expand=1#diff-27910c0ed3e4d6d075211ffc4db6cab4R1559

### CODE REVIEW

**Comment by snopoke** [comment]
@dannyroberts

**Comment by dannyroberts** [comment]
@snopoke reviewing, but the PR's not mergeable

**Comment by snopoke** [comment]
@dannyroberts fixed conflict

**Comment by dannyroberts** [comment]
Excellent

File: corehq/apps/app_manager/tests/data/suite/child-module-entry-datums.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<partial>
```
Comment (dannyroberts) [inline corehq/apps/app_manager/tests/data/suite/child-module-entry-datums.xml]: What's this file for? I can't see/find any references to it

Comment (dannyroberts) [inline corehq/apps/app_manager/tests/data/suite/child-module-entry-datums.xml]: Never mind, found it below w/o '.xml' suffix

**Pull request merged**